scripts/apps/archive/controllers/UploadController.ts

```diff
@@ -315,7 +322,7 @@ export function UploadController(
                 } else if (file.error.isAllowedFileType === false) {
                     uploadOfDisallowedFileTypesAttempted = true;
                 } else {
-                    invalidFiles.push(file.error.filename);
+                    invalidImages.push(file.error);
```

Comment:
How do you know that `file` is an image at this line? `result` can contain audio and videos too.

Reply:
I was returning the error object in case of images only.